fixes #20450 - change pagination to patternfly style

ohadlevy opened the pull request.

ohadlevy committed:
fixes #20450 - change pagination to patternfly style

rabajaj0509 commented:
LGTM 👍

ohadlevy commented:
@lzap any idea whats wrong with the tests?

lzap commented:
Will take a look shortly.

lzap commented:
Tests finally fixed [test].

lzap commented:
Please rebase on develop first.

ohadlevy commented:
@lzap rebased.

lzap commented on app/controllers/discovered_hosts_controller.rb:
```diff
@@ -336,4 +336,8 @@ def add_custom_facts
+
+  def resource_class
+    Host::Discovered
+  end
```
This should be out of `private` section.

lzap replied:
@ohadlevy you still have this in `private` section, I believe this causes all tests to fail.

lzap commented:
Then rebase please and tests should be green. Well, they randomly fail in core currently.

lzap commented:
Ping, what's above please, low effort.

ohadlevy commented:
@lzap rebased.

ohadlevy commented:
bump?

lzap reviewed (changes requested):
Tests are failing because of encapsulation there.

ohadlevy commented:
@lzap I've rebased and updated based on your comments, but it seems that the test failures indicate at existing failure in discovery (or its fixtures)? can you please take it over from here?

lzap commented:
The resource_class breaks all the tests, the controller is a hack around Edit Form and that's the reason - did not dig into details. I don't have time to take over, feel free to pick up this one.

ohadlevy commented:
@rahulbajaj0509 do you feel like picking this one up? I don't have time to work on this atm.
